.github/workflows/main.yml

```diff
@@ -0,0 +1,78 @@
+name: main builder
```

Comment:
If this file's name changed to `main.yml`, you need to update the reference at README.md(line 8) too. Also notice the README_zh.md